pyproject.toml

```diff
@@ -45,6 +45,7 @@ altair = { version = "*", optional = true }
 bokeh = { version = "*", optional = true }
 freezegun = { version = "*", optional = true }
 matplotlib = { version = "*", optional = true }
+seaborn = { version = "*", optional = true }
```

Comment:
We have to add it as a part of `dev` group as well (see lines between 112 and 143) :wink:. This works as seaborn is probably a dependency of some of the other packages.